model-server/src/test/kotlin/org/modelix/model/server/ReplicatedRepositoryTest.kt

```diff
@@ -63,7 +72,10 @@ class ReplicatedRepositoryTest {
             install(WebSockets)
             ModelReplicationServer(InMemoryStoreClient()).init(this)
         }
-        block()
+
+        coroutineScope {
+            block(this)
+        }
```

Comment:
Can this be replaced with [`runTest`](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-test/kotlinx.coroutines.test/run-test.html)?